.github/workflows/build.yml

```diff
@@ -537,10 +506,7 @@ jobs:
         log-level:
           - Information
           - Verbose
-        ntasks:
-          - 100
-          - 1000
-          - 1500
```

Comment:
I'd keep the `ntasks` matrix, even if it has only one value it will make easier to customize the test in the future if needed

Reply:
I removed it to reduce the number of jobs. They are now executed in the same job. Although the 1000 is only executed in Information log level